README.md

```diff
@@ -21,6 +21,30 @@ If you want the automatic approach, you can find a docker image [here](https://h
 If you want the ansible approach, you can find ansible role [here](https://github.com/misi/mm-ansible/).
 [![asciicast](https://asciinema.org/a/311365.svg)](https://asciinema.org/a/311365)
 
+## Package Installation
+If you want to install it on the Debian & Ubuntu based operating systems.
+* Prerequisites:
+multiparty-meeting will run on nodejs v12.x and later versions.
```

Comment:
We have an issue on node 12.x, that we don't have on 10.x so let's write 10.x on 12.x unfortunately we experience some very rarely occurring random crash.

Reply:
Alright, I've fixed it :)